docs/installation.rst

```diff
@@ -11,7 +11,46 @@ provided with pulpcore.
 Install ``pulp_rpm``
 --------------------
 
-Users should install from **either** PyPI or source.
+Users should install from **either** PyPI or source or use ansible-pulp installer.
+
+Install by Ansible-pulp
+***********************
+
+With use of ``ansible-pulp`` installer you need to download supportive
+role before you run installer.
+Only Fedora 29+ and CentOS 7 (with epel repository and python36) are supported.
+
+.. code-block:: bash
+
+   git clone https://github.com/pulp/ansible-pulp.git
+   cd ansible-pulp
+   ansible-galaxy install pulp.pulp_rpm_prerequisites -p ./roles/
+
+Then use role you downloaded **before** ansible-pulp installer roles.
+
+.. code-block:: yaml
+
+   ---
+   - hosts: all
+     vars:
+       pulp_secret_key: secret
+       pulp_default_admin_password: password
+       pulp_install_plugins:
+         pulp-rpm:
+           app_label: "rpm"
+     roles:
+       - pulp.pulp_rpm_prerequisites
+       - pulp-database
+       - pulp-workers
+       - pulp-resource-manager
+       - pulp-webserver
+       - pulp-content
+     environment:
+       DJANGO_SETTINGS_MODULE: pulpcore.app.settings
+
+Now you can run installer against your desired host following instructions
+in ansible-pulp installer.
```

Comment:
s/in/in the/
Also can this become a link to the installer be added here? Maybe here? https://github.com/pulp/ansible-pulp#using-the-example-playbook